adapter/scripts/codelldb/interface.py

```diff
@@ -239,10 +257,9 @@ def handle_message(body_ptr, body_len):
     return True
 
 
-@CFUNCTYPE(c_bool, py_object)
+@CFUNCTYPE(None, py_object)
```

Comment:
I made this change because the type signature for `drop_pyobject` returns `void`.
```
drop_pyobject: unsafe extern "C" fn(obj: *mut c_void)
```